docs/source/examples/workspaces/azure/PinFile

```diff
@@ -156,7 +156,7 @@ azure_vm_test:
         resource_group_type: "azure"
         resource_definitions:
           - vm_name: "{{ distro }}-TestAzureAuto"
-            resource_group: "{{ distro }}-LinchpinCreatedGroup"
+            resource_group: ccit
```

Comment:
Did you want to leave in `{{ distro }}` here to prevent distro conflicts in linchpin tests?

Reply:
One of the changes in the testing process, is to stop creating and destroying a resource group during testing. Not that it is a bad testing case, it is just that power is only given to service principle authentication profile, and it has too much authority withing our Azure organization. When this change will be merged, I will work on limiting the service principle account for only one resource group. The resource group used now for Linchpin testing is 'ccit'.